### ISSUE TITLE
Increase width of select values in issues page

### ISSUE THREAD

**Issue by iquiw** [akabekobeko/redmine-theme-minimalflat2#118]
The following is issues page in minimalflat2 theme.  All texts of `select` type values are not displayed.

![minimalflat2](https://user-images.githubusercontent.com/238681/32403900-6f79e42c-c188-11e7-8f7f-a317ab4f12a4.png)

The following is the same page in Default theme. The layout is not so good, but all texts are displayed. It seems Default theme does not specify `max-width`.

![defaulttheme](https://user-images.githubusercontent.com/238681/32403899-6eaf3fc4-c188-11e7-9d62-76b949891da8.png)

Can you consider to increase `max-width` of `fieldset#filters td.values select` ? Or is there any other better option?

**Comment by akabekobeko** [comment]
Its CSS is Redmine v3.4.2

```css
fieldset#filters td.values select {min-width:130px; max-width:200px;}
```

Although it was, in v3.4.3

```css
fieldset#filters td.values select {min-width:130px;}
```

It seems that it is becoming. Delete `max-width` as shown in v3.4.3.

**Comment by iquiw** [comment]
Oh, I have not aware that it was reported in Redmine too.

[Defect #26568 Multiple Selection List Filter View - items are cut off from view](http://www.redmine.org/issues/26568)

**Comment by akabekobeko** [comment]
@iquiw 

I fixed it with minimalflat2 v1.3.5. Please check it.

**Comment by iquiw** [comment]
Confirmed. Thank you!
